xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-ui/src/main/resources/Main/SolrSearchMacros.xml

```diff
@@ -824,6 +824,11 @@ $value##
 
     #end
     #displaySearchForm()
+    #foreach ($extension in $services.uix.getExtensions("org.xwiki.platform.search.widget"))
```

Comment:
It's ok. It would be great if you guys can take care of this decision. I will then modify my extension to make it compatible with the integrated uiextension point